packages/@react-stately/layout/src/TableLayout_DEPRECATED.ts

```diff
@@ -96,7 +96,7 @@ export class TableLayout_DEPRECATED<T> extends ListLayout<T> {
         let props = column.props as ColumnProps<T>;
         let minWidth = props.minWidth != null ? this.parseWidth(props.minWidth) : 75;
         let maxWidth = props.maxWidth != null ? this.parseWidth(props.maxWidth) : Infinity;
-        let width = Math.max(minWidth, Math.min(maxWidth, columnWidth));
```

Comment:
This is one legitimate change i'd like to make to the "old" code. Right now tests are relying on partial pixel values to pass. Rather than change tests to pass for both the feature-flagged and non-feature-flagged version, I do think it makes sense to keep the widths as whole pixels and not allow for partial pixels.

I believe Rob said they noticed a visual bug with dividers (most likely cause by partial pixels) and so this should address that and keep everything consistent between the old and new.